hack/update-manifests.sh

```diff
@@ -4,7 +4,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SRCROOT="$( CDPATH='' cd -- "$(dirname "$0")/.." && pwd -P )"
+SRCROOT="$(git rev-parse --show-toplevel)"
```

Comment:
Bad habit of mine changing things not knowing where they are used 😄

Is this change breaking? I can revert it.

Reply:
Tests pass, I say ship it. :-)